### ISSUE TITLE
Invalid org token (http status: 401)

### ISSUE THREAD

**Issue by rdeo-aziwell** [getsentry/sentry-android-gradle-plugin#694]
### Gradle Version

8.6

### AGP Version

8.3.2

### Code Minifier/Optimizer

R8

### Version

4.4.1

### Sentry SDK Version

7.8

### Steps to Reproduce

Using this configuration:
```kotlin
sentry {
  debug = true
  authToken = "sntrys_MYTOKEN"
  org = "MYORG"
  projectName = "MYPROJ"
}
```

### Expected Result

Obfuscation mappings get uploaded.

### Actual Result

I get this error:
```
> Task :app:uploadSentryProguardMappingsRelease FAILED
API request failed

  caused by: sentry reported an error: Invalid org token (http status: 401)
```
The token was generated from https://org.sentry.io/settings/auth-tokens/new-token/

**Comment by romtsn** [comment]
hey, I was able to reproduce the issue, it seems like we have to update our bundled sentry-cli version to support the EU data region. I'll roll out a fix shortly

**Comment by romtsn** [comment]
Closed by #684

**Comment by romtsn** [comment]
hi @rdeo-aziwell we've just shipped version 4.5.0, please give it a try and let us know if the issue is fixed. Thank you!
